lib/controllers/skill-code-controller/code-builder.js

```diff
@@ -57,22 +56,10 @@ class CodeBuilder {
                 callback(fsErr);
             });
         }
-        // 3.execute packge builder script
+        // 3. handle build
         if (this.buildFlow !== ZIP_ONLY_BUILD_FLOW) {
-            let command;
-            if (this.osType === 'WINDOWS') {
-                const doDebug = this.doDebug ? '$True' : '$False';
-                command = `${POWERSHELL_PREFIX} "& {& '${this.packageBuilder}' '${this.build.file}' ${doDebug} }"`;
-            } else {
-                command = `${this.packageBuilder} "${this.build.file}" ${!!this.doDebug}`;
-            }
-            shell.exec(command, { async: true, cwd: this.build.folder }, (code) => {
-                if (code !== 0) {
-                    callback(`[Error]: Build Scripts failed with non-zero code: ${code}.`);
-                } else {
-                    callback();
-                }
-            });
+            const { handler } = this.buildFlows[this.buildFlow];
+            this[handler]((error) => callback(error));
```

Comment:
this part also need try-catch since _build{language} function is calling child_process, but that process might fail as well. For instance user doesn't have "mvn" or "python3" installed in the system, we need to show the clear message.